disable_odoo_online/views/ir_ui_menu.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record model="ir.ui.menu" id="base.menu_module_updates">
```

Comment:
Place id attribute before model, according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#xml-files